packages/x-components/static-docs/build-search-ui/README.md

```diff
@@ -43,24 +45,25 @@ numerous components to choose from, and the catalog evolves quickly with new exp
 Check out the **[open source project in GitHub](https://github.com/empathyco/x)**.
 
 Interested in learning more about how Interface&nbsp;X works? Discover more about its
-**[architecture](x-architecture/README.md)**.
+**[architecture](web-x-architecture.md)**.
 
 ## Integrate Interface X Archetype
```

Comment:
Should this section be merged with the one below at  **Develop via Interface X Archetype**? It might be weird to explain how to integrate the solution before explaining how to develop the solution.

Reply:
I totally agree that it's confusing to place the integration doc in the first position when there's a previous step. However, Gerardo wanted it that way... in this file, as it's like a getting started doc, we can change the order and talk first about the development and the integration afterwards. What do you think @herrardo?

Additionally, from my non-tech point of view, it's also weird to have two development types (archetype & library) but just one dedicated integration.

Reply:
@javieri-empathy I understand your concern, but the thing here is that most of the people reading this section I think is going to be customers that didn't develop the Archetype instance(because we did) and they want just to integrate it on their website. That's why i wanted to put that at the beggining, to not make them feel they are losing something previous that they should read...does it make sense?

Reply:
You should take into account that the order in which present the pages in the sidebar doesn't need to match with the one presented in the getting started (readme). IMHO, from my own perspective, the structure may work as follows:

**For the Readme/Getting started:**
* Discover the ecosystem
* Develop using Interface X
  * Develop using the Archetype
  * Develop using the library
* Integrate the Archetype
* Use and configure X Components  

**For the sidebar:**
* Develop using the Archetype
* Integrate the Archetype
* Develop using the library
* How to use X Components

Note: remember that we've defined for the future to have other independent pages as Style your UI and Translate the search experience to cover both the design system & localization

Reply:
Understood @herrardo! Makes sense. I'm still not sure this is the best solution, but at least I understand that there is a reason to be sorted that way. FMPOV, i don't see this as a blocker. Can we note this and test it with the first readers of the doc? We can always change it in the future.

@mlcorton So if I understood correctly the sidebar is 100% independent from the markdowns?  I think that is powerful for use cases as the one you propose, but dangerous if it can desync.

Reply:
@javieri-empathy The sidebar is generated automatically based on the folders/files included in `src` from `docs-empathy-documentation` repo. But you can configure the contents and order to show in the sidebar if needed. Check out https://searchbroker.atlassian.net/wiki/spaces/ED/pages/3515187219/Configure+the+automatic+sidebar